backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/CloneImageGroupVolumesStructureCommand.java

```diff
@@ -184,29 +185,56 @@ private void createImage(DiskImage image, int imageIndex) {
         runInternalActionWithTasksContext(ActionType.CreateVolumeContainer, parameters);
     }
 
+    /**
+     * Method's calculation logic is described by the following table.
+     *
+     * type             src type        dst type        vol format          initial size
+     * =======================================================================================
+     * active           file            file            qcow2               null
+     * active           block           file            qcow2               null
+     * active           file            block           qcow2               3 chunks
+     * active           block           block           qcow2               3 chunks
+     * ---------------------------------------------------------------------------------------
+     * internal         file            file            raw                 null
+     * internal         file            file            qcow2               null
+     * internal         file            block           raw                 null
+     * internal         file            block           qcow2               measure
+     * internal         block           block           raw                 null
+     * internal         block           block           qcow2               measure
+     *
+     * Notes:
+     *  1. "chunks" stands for {@link org.ovirt.engine.core.common.config.ConfigValues#VolumeUtilizationChunkInMB}
+     *  2. "measure" stands for {@link ActionType#MeasureVolume} command's return value.
+     */
     private Long determineImageInitialSize(Image sourceImage,
-            VolumeFormat destFormat,
+            VolumeFormat volumeFormat,  // source volume format == destination volume format
```

Comment:
looking at [determineVolumeFormat](https://github.com/oVirt/ovirt-engine/blob/ovirt-engine-4.5.1.1/backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/CloneImageGroupVolumesStructureCommand.java#L248-L260) that sets this format, I see that the format of the destination volume might be different than that of the source volume - so I don't think the comment here is correct